Labs/Alyssa_Giedd_Lab4.ipynb

```diff
@@ -277,26 +277,289 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x237cb48c4a8>"
```

Comment:
I really like your 2D plots, I think they clearly show that the Earth longitude is the one that is not consistent over time. Definitely would be good to explain this and the differences between each plot!